conda/recipes/rapids/meta.yaml

```diff
@@ -60,6 +60,8 @@ requirements:
     - pylibcugraph ={{ minor_version }}.*
     - libcugraph_etl ={{ minor_version }}.*
     - ptxcompiler  # [linux64]  # CUDA enhanced compat. See https://github.com/rapidsai/ptxcompiler
+    - conda-forge::ucx-proc=*=gpu
+    - conda-forge::ucx {{ ucx_version }}
```

Comment:
Still seeing [the same issue]( https://gpuci.gpuopenanalytics.com/job/rapidsai/job/integration/job/prb/job/meta-pkg-nightly-build/1029/CONDA_CONFIG_FILE=conda%2Frecipes%2Fversions.yaml,CONDA_USERNAME=rapidsai-nightly,CUDA_VER=11.5,PYTHON_VER=3.9,RAPIDS_VER=22.06.00a/consoleFull ). So trying to add to the `rapids` metapackage.